protocols/user_data_processor.py

```
@@ -4,92 +4,107 @@
+                                )
+                            elif file.endswith(".txt"):
+                                processed_files.append(
+                                    {
+                                        "path": file_path.replace(
+                                            path, f"{name}/"
+                                        ),
+                                        "type": "text",
+                                        "size": size,
+                                    }
+                                )
                         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)